api/reservation_units_api.py

```diff
@@ -146,6 +146,11 @@ class ReservationUnitSerializer(TranslatedModelSerializer):
     unit_id = serializers.PrimaryKeyRelatedField(
         queryset=Unit.objects.all(), source="unit"
     )
+    unit_details = UnitSerializer(
```

Comment:
Actually we already have unit here in `ReservationUnitSerializer`- it was added just recently

Reply:
[here](https://github.com/City-of-Helsinki/tilavarauspalvelu-core/pull/462/files#diff-a6d1d83bc329f54c8f8a659daf844738e39aeb173daf244a794327e2578d608dR157)

Reply:
Ah, we seem to have had duplicate tickets created for this! I'll close this MR.